.env

```diff
@@ -0,0 +1,3 @@
+MONGOOSE_PORT=80
```

Comment:
AFAIK we decided to use another port for this outside of the container

Reply:
it's a port **inside** the container and it binds to a host port (38888 as an example in the picture below) by docker itself. 
Inside the container, NGINX is using this port (80). 
<img width="379" alt="image" src="https://user-images.githubusercontent.com/22101169/47189969-ce2d1580-d347-11e8-89e2-886d67f71ceb.png">

Reply:
Ok, but it should be "NGINX_PORT" and not "MONGOOSE_PORT"